framework/README.md

```diff
@@ -0,0 +1,23 @@
+# MuseScore Framework
+
+Framework (engine, platform) contains an independent domain modules for developing a MuseScore application.
+The framework includes modules such as:
+
+* Audio
+* Graphics
+* Fonts
+* UI Engine
+* Modularity
+* System
+* and other
+
+It’s like it’s a third-party engine, like Unity, Unreal Engine, etc.
```

Comment:
Sure about the spelling of those 2 `it's` here?

Reply:
Thanks, fixed